src/http/api/types.ts

```diff
@@ -119,24 +119,7 @@ export interface PostDTO {
     /**
      * Attachments of the post
      */
-    attachments: {
-        /**
-         * Type of the attachment
-         */
-        type: string;
-        /**
-         * Media type of the attachment
-         */
-        mediaType: string;
-        /**
-         * Name of the attachment
-         */
-        name: string;
-        /**
-         * URL of the attachment
-         */
-        url: string;
-    }[];
+    attachments: PostAttachment[];
```

Comment:
I think we should leave this as it was so that the API response is not dependent on the Entity representation (this means we can be flexible about whats returned where)